src/modules/peek/Peek.UI/Services/NeighboringItemsQuery.cs

```diff
@@ -27,12 +27,7 @@ public partial class NeighboringItemsQuery : ObservableObject
             IsMultipleFilesActivation = hasMoreThanOneItem;
 
             var neighboringItemsShellArray = hasMoreThanOneItem ? selectedItemsShellArray : FileExplorerHelper.GetItems(foregroundWindowHandle);
-            if (neighboringItemsShellArray == null)
-            {
-                return null;
-            }
-
-            return new NeighboringItems(neighboringItemsShellArray);
+            return neighboringItemsShellArray == null ? null : new NeighboringItems(neighboringItemsShellArray);
```

Comment:
nit: it's a a bit more readable when the null returned value is last in the ternary operator